### ISSUE TITLE
Unable to mount large external disk on Centos 7

### ISSUE THREAD

**Issue by mbach04** [relan/exfat#108]
```bash
[root@localhost ~]# mount.exfat-fuse -o ro /dev/sda /mnt
FUSE exfat 1.3.0
WARN: file system in sectors is larger than device: 250083328 * 512 > 127865454592.
ERROR: file system in clusters is larger than device: 976848 * 131072 > 127865454592.
```
Is there some means to bypass this warning and mount anyways? I understand the disk is much larger than the file system, but I'd still like to mount and read a file etc.

I'm also seeing an issue with unmount on a much smaller drive using the same version of exfat-fuse that causes the tty to hang completely. The session freezes completely.

**Comment by relan** [comment]
> Is there some means to bypass this warning and mount anyways? I understand the disk is much larger than the file system, but I'd still like to mount and read a file etc.

Only by editing the source code. This check is here because of some buggy embedded implementations (e.g. in cameras).

> I'm also seeing an issue with unmount on a much smaller drive using the same version of exfat-fuse that causes the tty to hang completely. The session freezes completely.

Please file a separate issues with more details.

**Comment by mbach04** [comment]
Yeah, prior to the issue submission I had done just that. I guess I was thinking that if this is something that only affects a small handlful of devices, then it would be best to have an option to ignore it, log the error to stderr and move forward with the mount.
